assets/js/components/settings/settings-module.js

```diff
@@ -127,7 +121,9 @@ class SettingsModule extends Component {
 	}
 
 	deactivate() {
-		if ( this.props.autoActivate ) {
+		const { slug } = this.props;
+
+		if ( 'search-console' === slug ) {
```

Comment:
@asvinb excuse me for confusing you, but the requirement to hardcode the SC module slug should have been deleted from the IB. :man_facepalming: Felix said that we couldn't hardcode it here and need to rely on the `forceActive` property of the module object. Please, revert this change to use `autoActivate` property and update the `SettingsActiveModules` component to pass `foceActive` value into SettingsModule elements as `autoActivate` property.